infoschema/tables_test.go

```diff
@@ -916,45 +919,67 @@ func (s *testTableSuite) TestForTableTiFlashReplica(c *C) {
 
 func (s *testClusterTableSuite) TestForClusterServerInfo(c *C) {
```

Comment:
It's better to move this test to the `executor` package.

Reply:
The test need `testClusterTableSuite` to setup rpcserver, move to `executor` copy the `testClusterTableSuite ` too.

I think it's ok here, since the table is from `information_schema`.